fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt

```diff
@@ -48,8 +50,18 @@ class EventBus {
      * i.e. synchronous.
      */
     fun fireEvent(event: Event) {
-        log.debug("Firing event: $event")
+        if (loggingEnabled) {
+            log.debug("Firing event: $event")
+        }
 
         eventHandlers.fireEvent(event)
     }
+
+    fun isLoggingEnabled(): Boolean {
```

Comment:
These two functions are not needed after the above change (removing `private`) is made.

Reply:
Fixed